content/en/platform/corda/4.10/enterprise/_index.md

```diff
@@ -4,17 +4,35 @@ menu:
   versions:
     weight: -648
   corda-enterprise-4-10:
+    identifier: about-corda-landing-4-10-enterprise
     weight: -30
-    name: Corda Enterprise Edition 4.10
+    name: Corda Enterprise 4.10
 project: corda
 section_menu: corda-enterprise-4-10
-title: Corda Enterprise Edition 4.10
+title: Corda Enterprise 4.10
 version: 'Enterprise 4.10'
 ---
 
-# Welcome to Corda Enterprise Edition 4.10
+# Corda Enterprise 4.10
+
+Corda is the world’s first private, permissioned distributed ledger technology (DLT) platform designed to work with today’s financial services industry. While regulated companies may start on public blockchains, they soon realize when they get to production, that they require capabilities native to Corda such as privacy, security, scalability, and ease-of-integration with existing systems. That’s why R3 is also exploring interoperability with assets that originate on a non-Corda network.
+
+A Corda Network is a peer-to-peer network of [Nodes](../../enterprise/node/component-topology.md), each representing a party on the network.
+These Nodes run Corda applications [(CorDapps)](../../enterprise/cordapps/cordapp-overview.html), and transact between Nodes using public or
+confidential identities.
+
+When one or more Nodes are involved in a transaction, the transaction must be notarised. [Notaries](../../enterprise/notary/ha-notary-service-overview.md) are a specialised type
+of Node that provides uniqueness consensus by attesting that, for a given transaction, it has not already signed other
+transactions that consumes any of the proposed transaction’s input states.
+
+To learn more about Corda, select any of the child pages of this **About Corda** section.
+
+## Corda Enterprise
```

Comment:
Doesn't seem like a great sub-heading? Similar to the H1 so not clear what the section contains?

Reply:
Yeah let me look - I already need to get rid of that "About Corda" mention too.

Reply:
OK removed that heading and moved paragraphs around to make it read better.